docs/en/04-supported-codebases/mmedit.md

```diff
@@ -1,20 +1,195 @@
+
+```shell
+git clone --recursive -b dev-1.x https://github.com/open-mmlab/mmdeploy.git
+cd mmdeploy
+python3 tools/scripts/build_ubuntu_x64_ort.py $(nproc)
+export PYTHONPATH=$(pwd)/build/lib:$PYTHONPATH
+export LD_LIBRARY_PATH=$(pwd)/../mmdeploy-dep/onnxruntime-linux-x64-1.8.1/lib/:$LD_LIBRARY_PATH
+```
+
+**Method III:** Build from source
+
+If neither **I** nor **II** meets your requirements, [building mmdeploy from source](../01-how-to-build/build_from_source.md) is the last option.
+
+## Convert model
+
+You can use [tools/deploy.py](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/tools/deploy.py) to convert mmedit models to the specified backend models. Its detailed usage can be learned from [here](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/02-how-to-run/convert_model.md#usage).
+
+When using `tools/deploy.py`, it is crucial to specify the correct deployment config. We've already provided builtin deployment config [files](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmedit) of all supported backends for mmedit, under which the config file path follows the pattern:
+
+```
+{task}/{task}_{backend}-{precision}_{static | dynamic}_{shape}.py
+```
+
+- **{task}:** task in mmedit.
+
+  MMDeploy supports models of one task in mmedit, i.e., `super resolution`. Please refer to chapter [supported models](#supported-models) for task-model organization.
+
+  **DO REMEMBER TO USE** the corresponding deployment config file when trying to convert models of different tasks.
+
+- **{backend}:** inference backend, such as onnxruntime, tensorrt, pplnn, ncnn, openvino, coreml etc.
+
+- **{precision}:** fp16, int8. When it's empty, it means fp32
+
+- **{static | dynamic}:** static shape or dynamic shape
+
+- **{shape}:** input shape or shape range of a model
+
+### Convert super resolution model
+
+The command below shows an example about converting `ESRGAN` model to onnx model that can be inferred by ONNX Runtime.
+
+```shell
+cd mmdeploy
+# download esrgan model from mmedit model zoo
+mim download mmedit --config esrgan_psnr-x4c64b23g32_1xb16-1000k_div2k --dest .
+# convert esrgan model to onnxruntime model with dynamic shape
+python tools/deploy.py \
+  configs/mmedit/super-resolution/super-resolution_onnxruntime_dynamic.py \
+  esrgan_psnr-x4c64b23g32_1xb16-1000k_div2k.py \
+  esrgan_psnr_x4c64b23g32_1x16_1000k_div2k_20200420-bf5c993c.pth \
+  demo/resources/face.png \
+  --work-dir mmdeploy_models/mmedit/ort \
+  --device cpu \
+  --show \
+  --dump-info
+```
+
+You can also convert the above model to other backend models by changing the deployment config file `*_onnxruntime_dynamic.py` to [others](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmedit), e.g., converting to tensorrt model by `super-resolution/super-resolution_tensorrt-_dynamic-32x32-512x512.py`.
+
+```{tip}
+When converting mmocr models to tensorrt models, --device should be set to "cuda"
```

Comment:
Shall we remove this tip for mmocr from mmedit?

Reply:
done